![](https://github.com/3d-omics/.github/blob/b345f29044584474a23b5bd5a58a8eda5a1f23ad/profile/3domics-logo.png
)

## Welcome to the 3D'omics code hub

The **3D'omics code hub** is the common space to develop, share and discuss about bioinformatic and statistical code used in 3D'omics. It contains multiple repositories dedicated to specific data processing and analysis steps. 3D'omics deals with two main types of omic data, conventional data generated at the macroscopic level (from faecal samples, gut contents, mucosal scrapes, etc.), and data generated at the micro-scale from microdissections. The multiple repositories stored in this hub are organised accordingly:

### Bioinformatics

#### Macro-scale multi-omics

* Host genomics
* Host transcriptomics
* Microbial genome-resolved metagenomics
* Microbial mapping-based metagenomics
* Microbial metatranscriptomics
* ...

#### Micro-scale multi-omics

* Microbial metagenomics
* Microbial metatranscriptomics
* ...

### Statistics

#### Macro-scale multi-omics

* ...

#### Micro-scale multi-omics

* ...

## What is 3D'omics?

3D'omics is a H2020 project that aims to reconstruct host-microbiota interactions in 3D at the micro-scale through a novel technology. Using poultry and swine production systems, we are analysing the effects of different factors, including animal development, diet, exposure to pathogens, and management practices on the so-called 3D omics landscapes of animal intestines. Further information about the project can be found in the [project website](https://www.3domics.eu/).

::::
This project has received funding from the European Unionâ€™s Horizon 2020 Research and Innovation programme under grant agreement number No. 101000309
:::
